dotcom-rendering/README.md

```diff
@@ -2,7 +2,21 @@
 
 Frontend rendering framework for theguardian.com. It uses [React](https://reactjs.org/), with [Emotion](https://emotion.sh) for styling.
 
-## Quick start
+## Quick start for first time dev install
+
+```sh
+$ corepack enable
+$ asdf reshim nodejs # if you're using asdf, this step is needed afterwards
+$ git clone git@github.com:guardian/dotcom-rendering.git
+$ nvm use # not needed if you're using asdf
```

Comment:
This means that this code block cannot be copy-pasted, because developers are unlikely to have two node management tools.

9. Isn't this only needed if you have just enabled corepack?

10. Should we really recommend the GitHub protocol for cloning? I thought we usually went for HTTPS? Is cloning the repo really something we need to bring attention to?

11. We generally recommend `fnm` over `nvm`, but even then I'm not here is the best place to write this out

---

I believe having `make dev` guide users through any requirements as it does now in the command line is more likely to be kept up to date than these docs, and would recommend we only write the following steps:

1. Clone the repository 
2. Enter the dotcom-rendering subdirectory `cd dotcom-rendering`
3. Start the dev server `make dev`